### ISSUE TITLE
Insufficient debug info when `Warning: The following definitions had errors while loading.`

### ISSUE THREAD

**Issue by dolezvo1** [Metacello/metacello#559]
I've been trying to update an older project to work with newer codebase. During the install, I got the following warning/error:
```
The following definitions had errors while loading.  Press Proceed to try to load them again (they may work on a second pass):
[list of classes with errors]
```
That is to be expected, but the issue is that this warning/error does not (as far as I can tell) provide any information about the individual errors in the given definitions, which is very unhelpful for someone trying to fix those errors.

There is a variable called `errorDefinitions` which contains the definitions mentioned in the message above, but no trace of the exact errors.

**Comment by Ducasse** [comment]
Yes we know and we faced this (suboptimal) design. It is bad. This behavior is a legacy. 
Can you tell us on which version of Pharo you are experiencing it?

**Comment by Ducasse** [comment]
Can you also provide a little case showing your problem so that we can use it to see how to fix it.
